packages/accordion/Accordion.tsx

```diff
@@ -1,29 +1,37 @@
-import React, { ForwardedRef, forwardRef } from 'react'
-import { AccordionProps } from './types'
-import {
-  AccordionStyle,
-  AccordionSummaryStyle,
-  AccordionTitleStyle,
-  AccordionArrowStyle,
-  AccordionContentStyle,
-} from './AccordionStyles'
+import { ForwardedRef, forwardRef } from 'react'
 import { useExpanded } from './useExpanded'
+import { NewLidoComponentProps } from 'packages/utils'
+import { ArrowBottom } from '../icons'
+import styles from './Accordion.module.css'
 
-function Accordion(props: AccordionProps, ref?: ForwardedRef<HTMLDivElement>) {
-  const { defaultExpanded, summary, children, ...rest } = props
-  const { toggleProps, collapseProps, isExpanded } = useExpanded(props)
+export type AccordionProps = NewLidoComponentProps<
+  'div',
+  {
+    defaultExpanded?: boolean
+    summary: React.ReactNode
+  }
+>
 
-  return (
-    <AccordionStyle {...rest} ref={ref}>
-      <AccordionSummaryStyle {...toggleProps}>
-        <AccordionTitleStyle>{summary}</AccordionTitleStyle>
-        <AccordionArrowStyle $expanded={isExpanded} />
-      </AccordionSummaryStyle>
-      <div {...collapseProps}>
-        <AccordionContentStyle>{children}</AccordionContentStyle>
-      </div>
-    </AccordionStyle>
-  )
-}
+export const Accordion = forwardRef(
+  (props: AccordionProps, ref?: ForwardedRef<HTMLDivElement>) => {
+    const { defaultExpanded, summary, children, ...rest } = props
+    const { toggleProps, collapseProps, isExpanded } = useExpanded(props)
 
-export default forwardRef(Accordion)
+    return (
+      <div {...rest} ref={ref} className={styles.accordion}>
+        <div {...toggleProps} className={styles.summary}>
+          <div className={styles.title}>{summary}</div>
+          <ArrowBottom
+            className={`${styles.arrow} ${
+              isExpanded ? styles.arrowExpanded : ''
+            }`}
```

Comment:
with `classnames` it will look like `cn(styles.arrow, {[styles.arrowExpanded]: isExpanded})`, consider bringing it to the project (it will be useful anyway)

Reply:
Makes sense, added